platform/ios/PolyPodApp/PolyPod.xcodeproj/project.pbxproj

```diff
@@ -797,6 +798,24 @@
 			shellPath = /bin/sh;
 			shellScript = "set -e\nENDPOINTS_SOURCE=\"$PROJECT_DIR/../../../../polyPod-config\"\nENDPOINTS_TARGET=\"$BUILT_PRODUCTS_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH/config\"\nrm -rf \"$ENDPOINTS_TARGET\"\ncp -r \"$ENDPOINTS_SOURCE\" \"$ENDPOINTS_TARGET\"\n";
 		};
+		519ACE372835887C00D56462 /* SwiftLint script */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "SwiftLint Script";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which swiftlint > /dev/null; then\n  swiftlint\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
Is it more likely that devs have Docker in a mac than having swiftlint installed? I say so because another possibility would be to simply use the docker image in the same way we do it in the workflow.

Reply:
Most iOS developers use SwiftLint this way. But, I know that SwiftLint can be also installed as Swift Package, thus it will not require external installation, I'll look more into it and hint to you @irstavr if it is possible to set up.